fritz

```diff
@@ -144,9 +141,27 @@ def run(args):
         with open("fritz.yaml") as fyaml:
             fritz_config = yaml.load(fyaml, Loader=yaml.FullLoader)
 
-        # add Fritz-specific K and SP extensions:
+        # add Fritz-specific K and SP extensions
+        # copy_tree('extensions/kowalski/', 'kowalski/')
         copy_tree('extensions/skyportal/', 'skyportal/')
 
+        # add Fritz-specific dependencies for SP
+        # js
+        with open("extensions/skyportal/package.fritz.json", "r") as fpjf, open("skyportal/package.json", "r") as fpj:
```

Comment:
Could also consider loading in each json into a dictionary.

Python 3 has the following dict merge syntax:

```
z = {**x, **y}
```

There's a dict-of-dict merge shown [on StackOverflow](https://stackoverflow.com/a/26853961/214686) that may be more suitable.

Oh, in fact, we have this built-in!

`baselayer.app.config.recursive_update`

Reply:
Sure, fixed